test/decorator.test.js

```diff
@@ -904,3 +904,38 @@ test('Request/reply decorators should be able to access the server instance', as
     t.equal(this.server.foo, 'bar')
   }
 })
+
+test('plugin required decorators', async t => {
```

Comment:
Could we add a test for the failure case?

I see only one that checks the `The decorator '${decorator}'${withPluginName} is not present in ${instance}` case

Reply:
That is already covered by current unit test